internal_cert_management/scripts/lego.sh

```
@@ -0,0 +1,67 @@
+
+# Validate required variables
+: "${ADMIN_EMAIL:?ADMIN_EMAIL is not set}"
+: "${SUBDOMAINS_EC256:?SUBDOMAINS_EC256 is not set}"
+: "${SUBDOMAINS_RSA2048:?SUBDOMAINS_RSA2048 is not set}"
+
+# Convert space-separated strings to arrays
+read -ra subdomains_ec256 <<< "$SUBDOMAINS_EC256"
+read -ra subdomains_rsa2048 <<< "$SUBDOMAINS_RSA2048"
+
+# Default types if not specified
+types=(rsa2048 ec256)
```

Shellcheck (0.11.0)

[warning] types appears unused. Verify use (or export if used externally).

(SC2034)